src/txn/txn_ckpt.c

```diff
@@ -106,7 +106,7 @@ __checkpoint_flush_tier(WT_SESSION_IMPL *session, bool force)
         cursor->get_value(cursor, &value);
         /* For now just switch tiers which just does metadata manipulation. */
         if (WT_PREFIX_MATCH(key, "tiered:")) {
-            __wt_verbose_debug2(
+            __wt_verbose(
```

Comment:
I think this message would be clearer if it said something like "force" or "no force" instead of "force 0" or "force 1". The concern is that somebody reading it wouldn't know that the 0/1 goes with the word "force" unless they look at the code.  (Unless, of course, this is a standard WT style that I just haven't picked up on.)

Alternatively, maybe just "force: 0" vs "force: 1" would do the trick.

There is a similar verbose message near the beginning of this function.

Reply:
How strongly do you feel about this? My only change was to lower the debug level so that I could always see the message. I expect that at some point in the future there will be a deep review of tiered verbose messages. The ones existing are strictly developer debugging. But I'm happy to make the change if you want it.

Reply:
It's not a big deal. You can leave it as is.